k8s/helm/prefect-agent/templates/secrets.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: prefect-config
+type: Opaque
```

Comment:
may want to make this optionally created.  some people may want to use an existing secret within their cluster